.github/workflows/ci.yml

```diff
@@ -453,4 +453,7 @@ jobs:
             nix run \
                 --accept-flake-config \
                 --extra-experimental-features '"nix-command flakes"' \
-                'github:flox/flox/${{ github.sha }}#packages.${{ matrix.system }}.flox-cli-tests' -- -- --filter-tags '"${{ matrix.test-tags }}"'
+                'github:flox/flox/${{ github.sha }}#packages.${{ matrix.system }}.flox-cli-tests' -- -- \
+                  --filter-tags '"${{ matrix.test-tags }}"' \
+                  --jobs 4 \
+                  --no-parallelize-across-files
```

Comment:
Can we add it to `flox-cli-tests` [yet](https://github.com/flox/flox/pull/2111#discussion_r1771180107)? 😁

Reply:
i still rather have control locally to enable parallel or bump it up to 10 jobs.
We also might consider not doing the parallel think on the remote builders since they seem easy to overload.

Reply:
Could we add `--jobs "${FLOX_BATS_JOBS:-4}"` in flox-cli-tests?

Reply:
added